src/main/java/com/powsybl/openloadflow/network/impl/LfNetworkLoaderImpl.java

```diff
@@ -70,26 +70,30 @@ public void visitThreeWindingsTransformer(ThreeWindingsTransformer transformer,
                     loadingContext.t3wtSet.add(transformer);
                 }
 
-                @Override
-                public void visitGenerator(Generator generator) {
+                private double checkVoltageRemoteControl(Injection injection, Terminal regulatingTerminal, double previousTargetV) {
                     double scaleV = 1;
-                    String controlBusId = generator.getRegulatingTerminal().getBusView().getBus().getId();
-                    String connectedBusId = generator.getTerminal().getBusView().getBus().getId();
+                    String controlBusId = regulatingTerminal.getBusView().getBus().getId();
+                    String connectedBusId = injection.getTerminal().getBusView().getBus().getId();
                     if (!Objects.equals(controlBusId, connectedBusId)) {
-                        if (generatorVoltageRemoteControl) {
+                        if (voltageRemoteControl) {
                             // remote control target bus will be set later because target bus might not have
                             // been yet created
-                            generatorRemoteControlTargetBusId.put(lfBus, controlBusId);
+                            voltageRemoteControlTargetBusId.put(lfBus, controlBusId);
                         } else {
-                            double previousTargetV = generator.getTargetV();
-                            double remoteNominalV = generator.getRegulatingTerminal().getVoltageLevel().getNominalV();
-                            double localNominalV = generator.getTerminal().getVoltageLevel().getNominalV();
+                            double remoteNominalV = regulatingTerminal.getVoltageLevel().getNominalV();
+                            double localNominalV = injection.getTerminal().getVoltageLevel().getNominalV();
                             scaleV = localNominalV / remoteNominalV;
-                            LOGGER.warn("Generator remote voltage control is not yet supported. The voltage target of " +
-                                            "generator {} with remote control is rescaled from {} to {}",
-                                    generator.getId(), previousTargetV, previousTargetV * scaleV);
+                            LOGGER.warn("Remote voltage control is not yet supported. The voltage target of " +
```

Comment:
Maybe change the comment just to say that remote voltage control is not active, so targets will be rescaled.

Reply:
Yes this warning has  been added while not yet implemented but now it has to be fixed "yet supported" => "active"